cmd/system-probe/subcommands/run/command.go

```diff
@@ -337,6 +338,7 @@ func startSystemProbe(log log.Component, statsd compstatsd.Component, telemetry
 			if pc := ebpftelemetry.NewPerfUsageCollector(); pc != nil {
 				telemetry.RegisterCollector(pc)
 			}
+			telemetry.RegisterCollector(ddebpf.NewLockContentionCollector())
```

Comment:
do we really need the debugPort configured for the contentionCollector?

Reply:
Since this is exposed as internal telemetry, I believe we do

Reply:
We can/should expose the telemetry over UDS too, but that would require some changes to the deployment charts I believe.